examples/ecs/ambiguity_detection.rs

```diff
@@ -0,0 +1,78 @@
+//! An example to confirm that `bevy` doesn't have system order ambiguity with [`DefaultPlugins`]
+//! This is run in CI to ensure that this doesn't regress again.
```

Comment:
Is this comment true?

Reply:
Yes, it is run in CI [through ci examples check](https://github.com/Vrixyz/bevy/blob/6ace0ed056773bed948c8b941ca520d02c24f348/.github/workflows/validation-jobs.yml#L144-L145) ; I believe I went through this because I failed at checking `RenderApp` with a simpler test.

I'll try again, maybe there's a way to test for RenderApp.

Reply:
seems to be working as an integration test / hidden example